charts/routes/values.yaml

```diff
@@ -172,6 +172,8 @@ mlpGateway:
   selector:
     istio: ingressgateway
 
+# configuration for extra mappings in VS (e.g. authentication proxies)
+extraMappings: []
```

Comment:
can we add 1/2 example mappings in the comment

Reply:
done